package.json

```diff
@@ -43,6 +43,7 @@
     "karma-sourcemap-loader": "0.3.5",
     "karma-webpack": "1.7.0",
     "lodash": "3.10.1",
+    "qunitjs": "^1.20.0",
```

Comment:
qunitjs was removed from deps because I started to use karma .... why did you add it back?